core-bundle/assets/backend.js

```diff
@@ -50,3 +50,6 @@ document.documentElement.addEventListener('turbo:frame-missing', (e) => {
     e.preventDefault();
     e.detail.visit(e.detail.response);
 });
+
+/* Styles */
+import './styles/backend.pcss';
```

Comment:
Is it technically necessary that the import is at the end of the file? If not, I would prefer it to be where the other imports are.

Reply:
Moved to the top in 230eabb7adbea0e91804553128b11ddd6ab03d02.